packages/app-store/routing-forms/lib/evaluateRaqbLogic.ts

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { Utils as QbUtils, type JsonTree } from "react-awesome-query-builder";
+import jsonLogic from "./jsonLogicOverrides";
+import { safeStringify } from "@calcom/lib/safeStringify";
+
+export const evaluateRaqbLogic = ({
```

Comment:
Abstracted out the logic to evaluate JSONLogic with RaqbConfig. It is now used at one more place and will be used more and more e.g. we could use it for evaluating to whom we should send instant meeting notification based on attributes logic.

It can handle any type of query builder config and data. So, it supports both FormFields and Attributes